backend/ddang/src/main/java/com/ddang/ddang/chat/presentation/ChatRoomController.java

```diff
@@ -70,13 +74,37 @@ public ResponseEntity<ReadChatRoomResponse> readChatRoomById(
 
     @PostMapping("/{chatRoomId}/messages")
     public ResponseEntity<CreateMessageResponse> createMessage(
+            @AuthenticateUser final AuthenticateUserInfo userInfo,
             @PathVariable final Long chatRoomId,
             @RequestBody @Valid final CreateMessageRequest request
     ) {
-        final Long messageId = messageService.create(CreateMessageDto.of(chatRoomId, request));
+        final Long messageId = messageService.create(CreateMessageDto.of(userInfo.id(), chatRoomId, request));
         final CreateMessageResponse response = new CreateMessageResponse(messageId);
 
         return ResponseEntity.created(URI.create("/chattings/" + chatRoomId + "/messages/" + messageId))
                              .body(response);
     }
+
+    @GetMapping("/{chatRoomId}/messages/{lastMessageId}")
+    public ResponseEntity<List<ReadMessageResponse>> readAllByLastMessageId(
+            @AuthenticateUser final AuthenticateUserInfo userInfo,
+            @PathVariable final Long chatRoomId,
+            @PathVariable(required = false) final Long lastMessageId
+    ) {
+        final ReadMessageRequest readMessageRequest = new ReadMessageRequest(userInfo.id(), chatRoomId, lastMessageId);
+        final List<ReadMessageDto> readMessageDtos = messageService.readAllByLastMessageId(readMessageRequest);
+        final List<ReadMessageResponse> responses = readMessageDtos.stream()
+                                                                   .map(readMessageDto -> ReadMessageResponse.of(
+                                                                           readMessageDto,
+                                                                           calculateMessageOwner(readMessageDto, userInfo))
+                                                                   )
```

Comment:
```suggestion
                                                                           calculateMessageOwner(readMessageDto, userInfo)
                                                                   ))
```

괄호가 닫혔는지가 헷갈려 위와 같이 수정하면 좋을 듯합니다!